packages/storage/src/implementation/request.ts

```diff
@@ -291,7 +291,7 @@ export function makeRequest<T>(
 ): Request<T> {
   let queryPart = UrlUtils.makeQueryString(requestInfo.urlParams);
   let url = requestInfo.url + queryPart;
-  let headers = object.clone<Headers>(requestInfo.headers);
+  let headers = clone(requestInfo.headers);
```

Comment:
This could be `{...requestInfo.headers}` or `Object.assign({}, requestInfo.headers)`.

Reply:
I tried not to change anything that was working fine after my refactoring. If I remove these functions that can be replaced by native functions (and I want to), I'll just change all the packages in a single PR and get approval from everyone. So I'll leave these as is for now.

Reply:
Done in next PR.